### ISSUE TITLE
GET requests should not require a Content-Type header

### ISSUE THREAD

**Issue by keithbro** [mickeyn/PONAPI#62]
```
$ curl http://localhost:5000/artists
{"jsonapi":{"version":"1.0"},"errors":[{"detail":"{JSON:API} Missing Content-Type header","status":415}]}
```

This is a GET request and has no body, so a Content-Type header should not be required.

https://www.w3.org/Protocols/rfc2616/rfc2616-sec14.html#sec14.17

"The Content-Type entity-header field indicates the media type of the entity-body sent to the recipient..."

I'm trying to use PONAPI in conjunction with ember-data which doesn't send a Content-Type for GET requests. I can configure the adapter to do so but it would be nice if it worked out of the box.

**Comment by mickeyn** [comment]
thanks @keithbro, I see your point.
I will drop the hard requirement, but an `Accept` will still be necessary in this case
(ember-data should be sending it)

**Comment by mickeyn** [comment]
I have made the fix and released it under PONAPI-Server 0.2.7.

**Comment by keithbro** [comment]
Amazing, thanks for doing it so quickly!
